code/modules/unit_tests/json_savefile_importing.dm

```diff
@@ -0,0 +1,48 @@
+/datum/unit_test/json_savefiles
+	var/savefile/test_savefile
+	var/json_savefile/json_savefile
+
+	var/list/basic_list
+	var/list/assoc_list
+	var/var_string
+	var/var_number
+
+/datum/unit_test/json_savefiles/New()
```

Comment:
Your New() should not be doing this much in the test. This should all be in Run().

Reply:
`You may use /New() and /Destroy() for setup/teardown respectively`

Reply:
I know I wrote that but this is really excessive setup